src/Core/Managed/Shared/TelemetryClient.cs

```diff
@@ -351,6 +351,12 @@ public void Track(ITelemetry telemetry)
                     return;
                 }
 
+                // If someone created configuration from scratch and forgot to initialize channel - use default
+                if (this.configuration.TelemetryChannel == null)
+                {
+                    this.configuration.TelemetryChannel = new InMemoryChannel();
+                }
+
```

Comment:
This works but now they have to pay the extra dereference and comparison check on every call.  It's not much of a cost but it is not free either.  They also won't know they didn't configure it officially since it will silently work with a different channel than maybe they were expecting.  If this is our official position then I am okay with it.